core/src/test/java/org/elasticsearch/cluster/metadata/WildcardExpressionResolverTests.java

```diff
@@ -128,5 +128,17 @@ public void testAll() {
     private IndexMetaData.Builder indexBuilder(String index) {
         return IndexMetaData.builder(index).settings(settings(Version.CURRENT).put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1).put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0));
     }
+    
+    public void testForDeprecatedWildCard() {
```

Comment:
it is not the wildcard that's deprecated, rather the '+' . Could you rename this method accordingly please?